data/36kr

```diff
@@ -0,0 +1,2 @@
+36kr.com
```

Comment:
You may want to mark `adx.36kr.com` as advertisement / trackers.

Reply:
also i noticed this website used a tracker script from `static.sensorsdata.cn`. 
that may be marked as tracker as well.

Reply:
Thanks. I'll have a rest first then make these changes.